src/pages/ClusterPage/NewMyCluster.js

```diff
@@ -480,7 +482,22 @@ class NewMyCluster extends Component {
 												color: 'red',
 											}}
 										>
-											{urlErrorMessage}
+											{urlErrorMessage === 'Auth Error' ? (
+												<React.Fragment>
+													We received a authentication error. Does your
+													ElasticSearch require additional authentication?
+													Read more{' '}
+													<a
+														target="_blank"
+														rel="noopener noreferrer"
+														href="https://docs.appbase.io"
```

Comment:
What link should we redirect to ?

Reply:
@lakhansamani should add this and he should ideally be the reviewer here.